x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/protections/memory.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+import { APP_ID } from '../../../../../../../common/constants';
+import { SecurityPageName } from '../../../../../../app/types';
+import { Immutable, OperatingSystem } from '../../../../../../../common/endpoint/types';
+import { MemoryProtectionOSes, OS } from '../../../types';
+import { ConfigForm } from '../../components/config_form';
+import { LinkToApp } from '../../../../../../common/components/endpoint/link_to_app';
+import { RadioButtons } from '../components/radio_buttons';
+import { UserNotification } from '../components/user_notification';
+import { ProtectionSwitch } from '../components/protection_switch';
+
+/** The Memory Protections form for policy details
+ *  which will configure for all relevant OSes.
+ */
+export const MemoryProtection = React.memo(() => {
+  const OSes: Immutable<MemoryProtectionOSes[]> = [OS.windows];
+  const protection = 'memory_protection';
+  return (
+    <ConfigForm
+      type={i18n.translate('xpack.securitySolution.endpoint.policy.details.memory_protection', {
+        defaultMessage: 'Memory Protection',
+      })}
+      supportedOss={[OperatingSystem.WINDOWS]}
+      dataTestSubj="memoryProtectionsForm"
+      rightCorner={<ProtectionSwitch protection={protection} osList={OSes} />}
+    >
+      <RadioButtons protection={protection} osList={OSes} />
+      <UserNotification protection={protection} osList={OSes} />
+      <EuiSpacer size="m" />
+      <EuiCallOut iconType="iInCircle">
+        <FormattedMessage
```

Comment:
I'm not sure of this, thus the question. Does this message/link need to be shown here? How many sections on the Policy Details UI are already showing this callout? if more than one, perhaps run it past Bonnie and Caitlin

Reply:
There is on-going work to redesign this section. I'll keep the repeated message for now as it will  change in the short-future.